packages/sdk-browser/CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # @turnkey/sdk-browser
 
+## 4.0.0
+
+### Major Changes
+
+- d1083bd: InitOtpV2 allows alphanumeric boolean and otpLength (6-9) to be passed + associated updates to EWK. More details below
+
+- This release introduces the `INIT_OTP_AUTH_V2` activity. The difference between it and `INIT_OTP_AUTH` is that it can now accept `alphanumeric` and `otpLength` for selecting crockford bech32 alphanumeric codes and the length of those codes. By default alphanumeric = true, otpLength = 9
+
+- This release introduces `send_from_email_sender_name` to `INIT_OTP_AUTH`, `INIT_OTP_AUTH_V2`, `EMAIL_AUTH` and `EMAIL_AUTH_V2`. This is an optional custom sender name for use with sendFromEmailAddress; if left empty, will default to 'Notifications'.
```

Comment:
Should we camel case this (in sdk-server changelog as well)? Given the assumption that the parameter is camelcased in JS land

Reply:
And to be consistent with `sendFromEmailAddress`

So in all, maybe something like

This release introduces `sendFromEmailSenderName` to the following activities: `INIT_OTP_AUTH`, `INIT_OTP_AUTH_V2`, `EMAIL_AUTH` and `EMAIL_AUTH_V2`. This is an optional custom sender name, for usage alongside `sendFromEmailAddress`. These parameters allow for the customization of email delivery. If left empty, `sendFromEmailSenderName` will default to 'Notifications'.